tests/frontends/torch/test_conv_transpose/test_torch_conv1d_transpose.py

```diff
@@ -0,0 +1,43 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import pytest
+import torch
+from hidet.testing.torch_utils import check_module
+import torch.backends.cudnn as cudnn
+
+
+@pytest.mark.parametrize('in_shape', [(1, 3, 224)])
+@pytest.mark.parametrize('w_shape', [(42, 3, 7)])
+@pytest.mark.parametrize('stride', [4])
+@pytest.mark.parametrize('padding', [3])
+@pytest.mark.parametrize('output_padding', [3])
+@pytest.mark.parametrize('groups', [1])
+@pytest.mark.parametrize('dtype', [torch.float32])
+def test_conv1d_transpose(in_shape, w_shape, stride, padding, output_padding, groups, dtype):
+    check_module(
```

Comment:
add 
```
cudnn.allow_tf32 = False
```

Reply:
To make it consistent, could you add the pairs of
```
cudnn.allow_tf32 = False 
```
and
```
cudnn.allow_tf32 = True 
```
to all conv/conv_transpose test?